README.md

```diff
@@ -2,7 +2,7 @@
 
 This repository provides a simple-to-use automated script to install the Amazon AVS SDK on a Raspberry Pi and configure the Raspberry Pi to use the **xCORE VocalFusion Kits for Amazon AVS**.
 
-The XMOS **xCORE VocalFusion Kits for Amazon AVS** provide far-field voice capture using the XMOS XVF3100, XVF3500, and XVF3510 voice processors.
+The XMOS **xCORE VocalFusion Kits for Amazon AVS** provide far-field voice capture using the XMOS XVF3100, XVF3500, XVF3510 and XVF3610 voice processors.
```

Comment:
Please add the XVF3000 to any list that also contains the XVF3100.  They're very nearly identical, but we sell them as separate products as far as I know.